packages/now-build-utils/src/detectors/filesystem.ts

```diff
@@ -58,84 +52,4 @@ export default abstract class DetectorFilesystem {
     }
     return p;
   };
-
-  public readJson = async <T>(name: string): Promise<T> => {
-    let p = this.readJsonCache.get(name);
-    if (!p) {
-      p = this.readFile(name).then(d => JSON.parse(d.toString('utf8')));
-      this.readJsonCache.set(name, p);
-    }
-    return p;
-  };
-
-  public readFileOrNull = async (name: string): Promise<Buffer | null> => {
-    return nullEnoent(this.readFile(name));
-  };
-
-  public readJsonOrNull = async <T>(name: string): Promise<T | null> => {
-    return nullEnoent(this.readJson<T>(name));
-  };
-
-  public readPackageJson = async (): Promise<PackageJson | null> => {
-    return await this.readJsonOrNull<PackageJson>('package.json');
-  };
-
-  public readConfigFile = async <T>(...names: string[]): Promise<T | null> => {
```

Comment:
Any reason why this was moved out of the DetectorFileSystem class?

Seems like we should either use it here or delete the whole file 🤔 

cc @TooTallNate

Reply:
@styfle yes, because it's now used in `@now/static-build` instead, and `DetectorFileSystem` doesn't require it anymore